src/main/java/teammates/ui/controller/InstructorFeedbackEditCopyAction.java

```diff
@@ -37,87 +39,96 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                 errorToAdmin += "Name of Session: " + newFeedbackSessionName + "<br>"; 
-                                               
-                errorToAdmin += "Copying to course(s) " + conflictCourse.toString() + " failed.";
+                errorToAdmin += "Copying to course(s) " + conflictCourses.toString() + " failed.";
                 log.severe(errorToAdmin);
-                                
-                String commaSeparatedListOfCourses = StringHelper.toString(conflictCourse, ",");
-                String errorToUser = String.format(Const.StatusMessages.FEEDBACK_SESSION_COPY_ALREADYEXISTS, newFeedbackSessionName, commaSeparatedListOfCourses);
                 
-                return createRedirectToEditPageWithErrorMsg(originalFeedbackSessionName, originalCourseId, errorToUser);
+                String commaSeparatedListOfCourses = StringHelper.toString(conflictCourses, ",");
+                String errorToUser = String.format(Const.StatusMessages.FEEDBACK_SESSION_COPY_ALREADYEXISTS,
+                                                   newFeedbackSessionName,
+                                                   commaSeparatedListOfCourses);
+                
+                return createRedirectToEditPageWithErrorMsg(originalFeedbackSessionName,
+                                                            originalCourseId,
+                                                            errorToUser);
             }
             
-            
             FeedbackSessionAttributes fs = null;
             // Copy the feedback sessions
             // TODO: consider doing this as a batch insert
             for (String courseIdToCopyTo : coursesIdToCopyTo) {
-                InstructorAttributes instructorForCourse = logic.getInstructorForGoogleId(courseIdToCopyTo, account.googleId);
-                gk.verifyAccessible(
-                        instructorForCourse, 
-                        logic.getCourse(courseIdToCopyTo), Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
+                InstructorAttributes instructorForCourse =
+                        logic.getInstructorForGoogleId(courseIdToCopyTo,
+                                                       account.googleId);
+                gk.verifyAccessible(instructorForCourse,
+                                    logic.getCourse(courseIdToCopyTo),
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
                 
-                fs = logic.copyFeedbackSession(newFeedbackSessionName, courseIdToCopyTo, originalFeedbackSessionName, originalCourseId, instructor.email);
+                fs = logic.copyFeedbackSession(newFeedbackSessionName,
+                                               courseIdToCopyTo,
+                                               originalFeedbackSessionName,
+                                               originalCourseId,
+                                               instructor.email);
```

Comment:
This is OK too. Although the indentation is a bit odd, the readability is preserved without adding too many new lines.
```
fs = logic.copyFeedbackSession(newFeedbackSessionName, courseIdToCopyTo,
                    originalFeedbackSessionName, originalCourseId, instructor.email);
```